addons/web/static/src/js/core/dom.js

```diff
@@ -89,13 +90,15 @@ return {
         minHeight = (options && options.min_height) || 50;
 
         $fixedTextarea = $('<textarea disabled>', {
-            class: $textarea[0].className,
-        }).css({
+            class: $textarea[0].className,});
+
+        var direction = _t.database.parameters.direction === 'rtl' ? 'right' : 'left';
```

Comment:
@mart-e Here it is used: https://github.com/odoo/odoo/pull/23831/files#diff-7bdbf63fc2c98509679e056e972c279aR101

Reply:
No, this actually does not work. You are setting the property `direction` with a value of `-10000` (which is ignored as invalid).

```js
> direction
"right"
> $fixedTextarea.css({direction: -10000})
> $fixedTextarea.css('right')
"auto"

Reply:
@mart-e ```direction``` is variable which contains value either ```left``` or ```right``` so value ```-10000``` is assigned to ```right```` or ```left```

Reply:
@msh-odoo A variable cannot be used as direct key of a javascript object... if you write `direction` there, it will be the `"direction"` string. To do what you want it is either like this:
```javascript
var props = {
    position: 'absolute',
    opacity: 0,
    height: 10,
    top: -10000,
};
props[direction] = -10000;
$fixedTextarea.css(props);
```
or like this:
```javascript
$fixedTextarea.css({
    position: 'absolute',
    opacity: 0,
    height: 10,
    top: -10000,
}).css(direction, -10000);
```

Note: if you use a linter as indicated here: https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines, you would have saw by yourself that the `direction` variable is unused here.

By the way... why is this needed anyway? The textarea here is a fake one which is put outside of the screen, does this have something to do with this task?